Total test count and pass count is wrong

![image](https://cloud.githubusercontent.com/assets/1427241/5417675/90d38872-8233-11e4-9958-1a00e1320c0a.png)

No idea why.

Sample output:
~> npm test

> test@1.0.0 test ~
> tape test.js

TAP version 13
# Simple test
ok 1 This is true
# Advanced test
not ok 2 ReferenceError: apple is not defined
  ---
    operator: ok
    expected: true
    actual:   false
    at: process._tickCallback (internal/process/next_tick.js:103:7)
  ...

1..2
# tests 2
# pass  1
# fail  1

npm ERR! Test failed.  See above for more details.
~> npm run color

> test@1.0.0 color ~
> colortape test.js

TAP version 13
# Simple test
ok 1 This is true
# Advanced test
not ok 2 ReferenceError: apple is not defined
  ---
    operator: ok
    expected: true
    actual:   false
    at: process._tickCallback (internal/process/next_tick.js:103:7)
  ...
TAP version 13

1..0
# tests 0
# pass  0

# ok

--- a/bin/colortape
+++ b/bin/colortape
@@ -2,7 +2,10 @@
 
 var path = require('path');
 var glob = require('glob');
-var test = require('tape');
+// To require the same tape module that is used by test files.
+var resolve = require('resolve');
+var tapePath = resolve.sync('tape', { basedir: process.cwd() });
+var test = require(tapePath);
 var ColorTape = require('..');
 var colorStream = new ColorTape();
 

--- a/package.json
+++ b/package.json
@@ -24,6 +24,7 @@
   "license": "BSD",
   "dependencies": {
     "glob": "^4.3.1",
+    "resolve": "^1.1.7",
     "tape": "^3.0.3"
   }
 }

--- a/package.json
+++ b/package.json
@@ -23,8 +23,8 @@
   "author": "Shuhei Kagawa <shuhei.kagawa@gmail.com>",
   "license": "BSD",
   "dependencies": {
-    "glob": "^4.3.1",
+    "glob": "^7.1.1",
     "resolve": "^1.1.7",
-    "tape": "^3.0.3"
+    "tape": ">= 3.0.0"
   }
 }

